blockchain/pallets/minting/src/lib.rs

```diff
@@ -192,7 +196,16 @@ pub mod pallet {
             IdDatasets::<T>::insert(id, extension_proof);
             NextMintingRewards::<T>::insert(id, who);
 
-            Ok(())
+            // Only first call is free, determined by not having an
+            // entry in .CalledRegisterForMinting. All subsequent calls
+            // will be Pays::Yes
+            if CalledRegisterForMinting::<T>::try_get(id).is_ok() {
```

Comment:
I think with this logic, I would need to pay a fee if I did the following:

register_identity(address1)
register_for_minting(address1)
register_identity(address2)
register_for_minting(address2)

I wouldn't be able to bootstrap address2 with the logic as is. Bootstrapping an address is the whole reason to not require a fee.

Reply:
Agree, this was misunderstood, I implemented only once in an absolute sense. Wat not aware of the requirement of bootstrapping a second address on the same fractal_id